### PR TITLE
chore(file_store): Document that it's a dev database

### PR DESCRIPTION
**Pull request by LLFourn**
Document `bdk_file_store` is a development/testing database

### CODE REVIEW

**Review by ValuedMammal — APPROVED** [review]
ACK 4b5bd0b14bfba95f0ca4e2c612d6e24832ba5b73

File: crates/file_store/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # BDK File Store
 
+> ⚠ `bdk_file_store` is a development/testing database. It does not natively support backwards compatible BDK version upgrades so should not be used in production.
```
Comment (oleonardolima) [inline crates/file_store/README.md:3]: I'd personally make it even clearer by using bold and uppercase.

```suggestion
> ⚠ `bdk_file_store` is a development/testing database. It does not natively support backwards compatible BDK version upgrades so *SHOULD NOT BE USED IN PRODUCTION*.
```

**Review by oleonardolima — COMMENTED** [review]
ACK 4b5bd0b14bfba95f0ca4e2c612d6e24832ba5b73

**Comment by LagginTimes** [comment]
ACK 4b5bd0b14bfba95f0ca4e2c612d6e24832ba5b73

**Pull request merged**